wallet/src/components/IbcTransfer.tsx

```diff
@@ -34,11 +34,11 @@ export enum IbcDirection {
 const unmodifiableAddressStyle = {
   width: 420,
   '& .Mui-disabled': {
-    color: 'rgba(0,0,0,0.6)',
+    color: 'rgba(0,0,0,0.6) !important',
```

Comment:
Danger will Robinson 😉 

This is pure tech debt. Please leave an XXX comment with justification so we don't normalize it

Reply:
Yea this is absolutely a last resort. I wanted to use a disabled text input for displaying addresses because it has all the layout and styling I need, but the default color is way too faint and hard to change.